lib/sidekiq/scheduler.rb

```diff
@@ -137,7 +137,9 @@ def self.handle_errors
     def self.enqueue_job(job_config)
       config = job_config.dup
 
-      config['class'] = config['class'].constantize if config['class'].is_a?(String)
+      if config['class'].is_a?(String) && Object.const_defined?(config['class'])
+        config['class'] = config['class'].constantize
+      end
       config['args'] = Array(config['args'])
 
       if defined?(ActiveJob::Enqueuing) && config['class'].included_modules.include?(ActiveJob::Enqueuing)
```

Comment:
If 'project 1' depends on ActiveJob but the job's class exists only in 'project 2', `config['class'].included_modules` will fail. I think that config['class'] type should be verified before asking for the included modules.